### ISSUE TITLE
Test Creating New Snapshot Proposal

### ISSUE THREAD

**Issue by masvelio** [hicommonwealth/commonwealth#3261]
Test every possible paths (different communities etc) for creating new proposal. Create tickets with bugs

**Comment by CowMuon** [comment]
Unblocked. Ryan's Quill work now merged into `react` branch.

**Comment by CowMuon** [comment]
Per my [other comment](https://github.com/hicommonwealth/commonwealth/issues/3236#issuecomment-1496764288), we have multiple bug reports apparently related to snapshot proposals across various communities. 

Now that's Ryan's [View Snapshot proposal page](https://github.com/hicommonwealth/commonwealth/pull/3332) is merged into React, let's go ahead and test proposal creation on the React branch across these communities and make sure proposal creation as well as viewing is just working end-to-end on the Reach branch. 



 @alexyoung23j to take the lead on this, pairing with @ianrowan and anyone else as needed -  @jnaviask @timolegros. We need to ship this on Wednesday 4/5. 
 
 I'm pretty sure we never reproduced the linked bugs, which were reported in at least 3 communities, which is not good. 
 
 Having said that, let's get some more hands on deck here to test this, on both branches, and confirm whether this bug is reproducible or not on either branch. @alexyoung23j @jackhmarcus @ianrowan

**Comment by alexyoung23j** [comment]
Addressed in #3236
